frost-ristretto255/tests/proptests.rs

```diff
@@ -40,36 +37,32 @@ impl SignatureCase {
     fn new<R: RngCore + CryptoRng>(mut rng: R, msg: Vec<u8>) -> Self {
         let sk = SigningKey::new(&mut rng);
         let sig = sk.sign(&mut rng, &msg);
-        let pk_bytes = VerificationKey::from(&sk).into();
-        let invalid_pk_bytes = VerificationKey::from(&SigningKey::new(&mut rng)).into();
+        let vk = VerifyingKey::from(&sk);
+        let invalid_vk = VerifyingKey::from(&SigningKey::new(&mut rng));
         Self {
             msg,
             sig,
-            pk_bytes,
-            invalid_pk_bytes,
+            vk,
+            invalid_vk,
             is_valid: true,
         }
     }
 
     // Check that signature verification succeeds or fails, as expected.
     fn check(&self) -> bool {
-        // The signature data is stored in (refined) byte types, but do a round trip
-        // conversion to raw bytes to exercise those code paths.
-        let sig = {
-            let bytes: [u8; 64] = self.sig.into();
-            Signature::from(bytes)
-        };
-        let pk_bytes = {
-            let bytes: [u8; 32] = self.pk_bytes.into();
-            VerificationKeyBytes::from(bytes)
-        };
-
-        // Check that the verification key is a valid RedJubjub verification key.
-        let pub_key = VerificationKey::try_from(pk_bytes)
-            .expect("The test verification key to be well-formed.");
+        // // The signature data is stored in (refined) byte types, but do a round trip
+        // // conversion to raw bytes to exercise those code paths.
+        // let sig = {
+        //     let bytes: [u8; 64] = self.sig.into();
+        //     Signature::<C>::from_bytes(bytes)
+        // };
+
+        // // Check that the verification key is a valid key.
+        // let pub_key = VerifyingKey::<C>::from_bytes(pk_bytes)
+        //     .expect("The test verification key to be well-formed.");
```

Comment:
Were these commented out on purpose? If yes, I suggest adding the reason as comment, or just deleting it

Reply:
Oh yeah I forgot this, I'll fixup

Reply:
Resolved in d849c281b6a89f74bbe8e96ecc518f9a7c49698d